docs/atlas-testnet.rst

```diff
@@ -0,0 +1,67 @@
+Atlas Testnet
+=============
+
+Atlas is a community testnet which can be used to test setting up a
+cosmos validator node. In addition to this tutorial you can also
+follow `this video <https://www.youtube.com/watch?v=B-shjoqvnnY>`__.
+
+Get Tokens
+----------
+
+To work on the Atlas you'll need some tokens to get started. To do
+this first generate a new key:
+
+::
+
+    gaia client keys new bob
```

Comment:
Should maybe just link to the first tutorial page to make the keys? shouldn't explain how to make the keys in a billion places